h/static/scripts/app.coffee

```diff
@@ -93,50 +76,81 @@ setupStreamer = [
     $http.defaults.headers.common['X-Client-Id'] = clientId
 ]
 
-module = angular.module('h', imports)
+module.exports = angular.module('h', [
+  'bootstrap'
+  'ngAnimate'
+  'ngResource'
+  'ngRoute'
+  'ngSanitize'
+  'ngTagsInput'
+  'toastr'
+])
+
 .config(configureDocument)
 .config(configureLocation)
 .config(configureRoutes)
 .config(configureTemplates)
 
-unless mocha? # Crude method of detecting test environment.
-  module.run(setupCrossFrame)
-  module.run(setupStreamer)
-  module.run(setupHost)
-
-require('./vendor/annotator.auth.js')
-require('./annotator/monkey')
-
-require('./controllers')
-
-require('./directives')
-require('./directives/annotation')
-require('./directives/deep-count')
-require('./directives/markdown')
-require('./directives/privacy')
-require('./directives/simple-search')
-require('./directives/status-button')
-require('./directives/thread-filter')
-require('./directives/thread')
-
-require('./filters')
-require('./searchfilters')
-
-require('./services')
-require('./annotation-mapper-service')
-require('./annotation-ui-service')
-require('./auth-service')
-require('./cross-frame-service')
-require('./host-service')
-require('./flash-service')
-require('./permissions-service')
-require('./local-storage-service')
-require('./store-service')
-require('./threading-service')
-
-require('./streamsearch')
-
-require('./annotation-sync')
-require('./annotation-ui-sync')
-require('./bridge')
-require('./discovery')
+.controller('AppController', require('./app-controller'))
+.controller('AnnotationController', require('./annotation-controller'))
+.controller('AnnotationUIController', require('./annotation-ui-controller'))
+.controller('StreamController', require('./stream-controller'))
+.controller('WidgetController', require('./widget-controller'))
+
+.directive('annotation', require('./directive/annotation'))
```

Comment:
Good work on getting all the directives consistent. I'll note that I've double checked we've got the same set of directives before and after :)